src/main/java/eu/europeana/thumbnail/web/ThumbnailController.java

```diff
@@ -116,21 +119,62 @@
 
         return result;
     }
+    /**
+     * Retrieves Header values
+     * @param url optional, the URL of the media resource of which a thumbnail should be returned. Note that the URL should be encoded.
+     *            When no url is provided a default thumbnail will be returned
+     * @param size optional, the size of the thumbnail, can either be w200 (width 200) or w400 (width 400).
+     * @param type optional, type of the default thumbnail (media image) in case the thumbnail does not exists or no url is provided,
+     *             can be: IMAGE, SOUND, VIDEO, TEXT or 3D.
+     * @return responsEntity
+     */
+    @RequestMapping(value = "/v2/thumbnail-by-url.json",
+            method = {RequestMethod.HEAD})
+    public ResponseEntity thumbnailByUrlHead(
+            @RequestParam(value = "uri") String url,
+            @RequestParam(value = "size", required = false, defaultValue = "w400") String size,
+            @RequestParam(value = "type", required = false, defaultValue = "IMAGE") String type,
+            WebRequest webRequest, HttpServletResponse response) {
+        long startTime = 0;
+        if (LOG.isDebugEnabled()) {
+            startTime = System.nanoTime();
+            LOG.debug("Thumbnail url = {}, size = {}, type = {}", url, size, type);
+        }
+        ResponseEntity result;
+        HttpHeaders headers= new HttpHeaders();
+        ControllerUtils.addResponseHeaders(response);
+        ObjectMetadata metadata = getMetaData(computeResourceUrl(url, size));
+
+        if(metadata!=null) {
```

Comment:
Our code convention is to add a space behind "if" and between the left and right side of the expression
For example: if (metadata != null)